docs/submit/osg-flock.md

```diff
@@ -0,0 +1,187 @@
+Configuring a submit host to flock to OSG
+=========================================
+
+Overview
+--------
+
+If you have a HTCondor submit node on your campus, it can be configured
+to spill over onto available resources on the Open Science Grid. In
+HTCondor terms this is called [flocking](https://research.cs.wisc.edu/htcondor/manual/latest/ConnectingHTCondorPoolswithFlocking.html)
+
+If you are interested in this solution, please open a
+[new ticket](https://support.opensciencegrid.org/helpdesk/tickets/new) with the hostname.
+
+Requirements
+------------
+
+The requirements are:
+
+* A public IP address, forward and reverse DNS.
+* Ability to open a few firewall incoming ports to the WAN.
+* HTCondor has to authenticate via pool GSI (preffered) or password. For GSI, the submit host
+   has to have a host certificate.
+* Reporting to the OSG accounting system has to be enabled. This can
+   be accomplished by installing and configuring the *gratia-probe-condor* and *gratia-probe-glideinwms* RPMs.
+* Submitted jobs should have the *+ProjectName* attribute specified with
+   a valid registered project name.
+
+
+Required Packages
+-----------------
+
+Enable the [OSG Yum repository](/common/yum/).
+
+Install the packages required:
+
+```console
+root@host # yum install osg-flock
+```
+
+Gratia Probe Configuration
+--------------------------
+
+Edit the `/etc/gratia/condor/ProbeConfig` file. Change the probe section to be:
+
+```file
+    ProbeName="condor:YOUR_HOSTNAME_HERE"
+    SiteName="YOUR_SITE_HERE"
+    Grid="OSG"
+    SuppressUnknownVORecords="0"
+    SuppressNoDNRecords="0"
+    SuppressGridLocalRecords="1"
+    QuarantineUnknownVORecords="1"
+    EnableProbe="1"
+
+    MapUnknownToGroup="1"
+    MapGroupToRole="1"
+    VOOverride="osg"
+```
+
+Please remember to enable and start the probe:
+
+```console
+root@host # systemctl enable gratia-probes-cron
+root@host # systemctl start gratia-probes-cron
+```
+
+GSI: HTCondor Configuration
+---------------------------
```

Comment:
@rynge since we ship this configuration in `osg-flock`, can we just get rid of this section?

Reply:
I think we should. Maybe structure it as osg-flock is strongly recommended, but if that can be done, use pool passwords?

Reply:
That makes sense. @efajardo, can you 

1. Get rid of this section
1. Change the pool password heading to indicate that it's optional
1. Write a sentence or two at the top of pool password section indicating that admins should only configure their schedd this way if they can't get a certificate
1. Tell them that they should replace the contents of `/etc/condor/config.d/80-osg-flocking.conf` with the config as described.

@rynge we could probably make this a little easier to config for admins: we can put the pool password config in an if block that triggers off the admin adding a config value like `OSG_FLOCKING_POOL_PASSWORD = True`